src/main/java/persistence/sql/ddl/clause/column/ColumnClauses.java

```diff
@@ -14,7 +15,7 @@ public ColumnClauses(List<Field> fields) {
 
     private static List<ColumnClause> getColumnClauses(List<Field> fields) {
         return fields.stream()
-                .filter(field -> !field.isAnnotationPresent(Transient.class))
+                .filter(field -> !field.isAnnotationPresent(Transient.class) && !field.isAnnotationPresent(OneToMany.class))
```

Comment:
```suggestion
                .filter(field -> !field.isAnnotationPresent(Transient.class))
                .filter(field -> !field.isAnnotationPresent(OneToMany.class))
```
이렇게 작성하는 편이 가독성이 더 좋은것 같은데 어떻게 생각하세요?